.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+profile=black
+known_first_party=ansiblelint
+known_third_party=ansible,pytest,ruamel,setuptools,sphinx,yaml
+# picked to match our current flake8 value
```

Comment:
it must be that value - 1

Reply:
We have custom value 100, and we want to be sure it matches it. If we are to debate line length it would be in separated change. I do not want isort to produce results that are not passing our flake8 config.

Reply:
I understand. The reason for this that in the past flake8 used to do `>` comparison while isort had `>=` (https://github.com/wemake-services/wemake-python-styleguide/commit/b20eb1a555194b1d19cffcd78821b99d73bb8f69#diff-380c6a8ebbbce17d55d50ef17d3cf906R66-R67). And this meant that isort setting should be lower than flake8 to render the same behavior.

It seems like it's been fixed since then, though.

FWIW setting a lower setting for isort is fully compatible with the flake8 setup as long as it has higher limit.